hub: refresh/restructure repo content (#21435)
docker/docs: content/manuals/docker-hub/repos/manage/vulnerability-scanning.md

---
description: Learn about vulnerability scanning and Docker Scout image analysis in Docker Hub.
keywords: scanning, vulnerabilities, Hub, static
title: Image security insights
weight: 70
aliases:
- /docker-hub/vulnerability-scanning/
---

Strengthen the security of your Docker images with Docker Hub's image security
insights. Docker Hub lets you perform either point-in-time static vulnerability
scanning or always up-to-date image analysis using Docker Scout.

## Docker Scout image analysis

> When you have an image in your Docker Hub repository after turning on Docker
> Scout image analysis, Docker Scout automatically analyzes the image.

Reviewer: ```suggestion
After turning on Docker Scout image analysis, Docker Scout automatically analyzes images in your Docker Hub repository.
```

Reviewer: just a rewording for clarity
